### DISCUSSION TITLE
Option to disable sending bee counts (Purpur Client)

### DISCUSSION THREAD

**Discussion by Potothingi** [PurpurMC/Purpur#1434] [__Completed__] [answered]
I don't want Purpur client players to see the number of bees. So, I suggest adding an option.
https://github.com/PurpurMC/Purpur/blob/ver/1.20.1/patches/server/0237-Give-bee-counts-in-beehives-to-Purpur-clients.patch

**Comment by Potothingi** [comment]
I don't want Purpur to send things to Purpur clients that the vanilla clients can't see.

**Comment by granny** [comment, marked as answer]
Implemented as of build #2146 https://github.com/PurpurMC/Purpur/commit/5b468f25626e796aee99ddef87509861a58780a7. It defaults to `false`.

**Comment by Potothingi** [comment]
I was already anticipating the exploit when I wrote this discussion.
